### ISSUE TITLE
88- QC Samples shouldn't be transferred

### ISSUE THREAD

**Issue by diegoliberman** [instedd/cdx#1479]
### Details
 As the QC Sample is the one that contains all the relevant assay information that sets the bases for all sample recipients to run their tests, these should not be allowed to be transferred. 

### Behaviour
- When users select samples from a list, the Transfer icon should not be enabled if a QC sample is selected
- A tooltip could be displayed on hover over the Transfer icon, when disabled if a Q sample is selected, to let the user know that to be able to perform the action, the QC can't be transferred.

### Questions
- What do we do if a user changes the QC Specimen Role from Q to anything else after the assays have been added, and then transfers the sample? Should we add a message to users warning them about the action they are performing? Should prompt them to select a different sample as Q and upload the assays to it? 
- Should we block Q samples Specimen Role once this is selected, to prevent changes that might impact negatively the UX and flow of the platform?
-

**Comment by diegoliberman** [comment]
On hold until we determine if this is OK for non-UCSD use cases.

**Comment by gbatiston** [comment]
Unblocked by decisions made around #1483 and #1484

**Comment by gbatiston** [comment]
Duplicates #1483
